api/models.py

```diff
@@ -89,3 +90,58 @@ def save(self, *args, **kwargs):
     def __str__(self):
         return '{}{}{}'.format(self.item_code, self.serial_number,
                                self.model_number)
+
+
+class UserManager(BaseUserManager):
+    use_in_migrations = True
+
+    def _create_user(self, **fields):
```

Comment:
let's also have tests for these methods. you can call `User.objects._create_user`, `User.objects.create_user`, `User.objects.create_superuser` and assert a user is created, and values in  `is_staff` and `is_superuser`

Reply:
I'm on it